src/app/modules/item/services/discussion.service.ts

```diff
@@ -28,10 +28,7 @@ export class DiscussionService implements OnDestroy {
   private configuredThread: ThreadId|null = null;
   private hasForcedThread = false;
 
-  /** combination of `canShowInCurrentPage$` and `visible$` which can be useful for some components */
-  state$ = combineLatest([ this.canShowInCurrentPage$, this.visible$ ]).pipe(
-    map(([ canShowInCurrentPage, visible ]) => (canShowInCurrentPage ? { visible } : undefined))
-  );
+  state$ = this.visible$.pipe(map(visible => ({ visible })));
```

Comment:
Arf don't break that, just use `visible` if you want only visible ! No need to wrap it in an object. 

That composition was very useful. The reason for this (maybe become unused for the moment but still make sense sooner or later) is that the 2 variables are not really independent... and as I explained to you, I hate non-independent variables 🤨😁. In practice, it is because "visible" made no sense if you were not allowed to show... but I understand that it may change now. 
But still, no need to wrap it. :-)